src/index.d.ts

```diff
@@ -32,3 +32,12 @@ export interface HeadingData {
  * @returns A string formatted the same as what would {@link gfmHeadingId} do.
  */
 export function getHeadingList(): HeadingData[];
+
+/**
+ * Returns a list of headings with the ids as computed by gfmHeadingId
```

Comment:
this comment should reflect what resetHeadings() does

Reply:
Revenge of the Cut-n-paste. :) Should be clearer.